Applications/WebPortal/WebApp/library/source/components/country_selection_box.tsx

```diff
@@ -2,7 +2,6 @@ import { css, StyleSheet } from 'aphrodite';
 import * as React from 'react';
 import * as Nexus from 'nexus';
 import { DisplaySize } from '../display_size';
-import { TextField } from '../pages/account_page/profile_page';
```

Comment:
```TextField``` should be imported from the components folder, not profile_page.

Reply:
That TextField has the wrong style so I can't use it for this.

Reply:
Okay take ```TextField```, rename it to ```TextInputField```, place that component in the components folder and style it to match the one defined in ui_kit/text_input_field.xd.

This way we only have one single ```TextInputField``` that's defined as part of the web portal's UI kit.

Reply:
There are two different TextFields defined in the UI kit. I'd rather the one we're adding to components be called `ProfileTextField` because the differences between them are pretty subtle and the `TextField `already in components is the one used most often.